src/OpenTK/Toolkit.cs

```diff
@@ -129,6 +132,18 @@ public static Toolkit Init(ToolkitOptions options)
                     initialized = true;
                     Configuration.Init(options);
                     Options = options;
+                    if (Environment.OSVersion.Platform == PlatformID.Win32S | Environment.OSVersion.Platform == PlatformID.Win32Windows | Environment.OSVersion.Platform == PlatformID.Win32NT)
```

Comment:
This this not use the logical or operator `||` instead?

Reply:
Can change if you want, especially as it's not in a performance critical area, although this one would be insignificantly faster, & I've done it this way out of habit :P

Reply:
It's not a performance concern it's a readability and correctness term. The precidence between `|` and `==` is not immediately obvious sand generally it's expected that is you are working with booleans you use logical operators instead of bitwise operators.